test/test_dynamicLeaderboard.py

```diff
@@ -0,0 +1,26 @@
+from testing_base import *
```

Comment:
This test looks good! Clean and shows full functionality of the updateLeaderboard function.